pkgs/development/libraries/tpm2-tss/default.nix

```diff
@@ -24,13 +24,23 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [
-    autoreconfHook autoconf-archive pkg-config doxygen perl
+    autoreconfHook autoconf-archive pkg-config doxygen perl procps_pkg iproute2 ibm-sw-tpm2
```

Comment:
I think these should stay in checkInputs, because they are only required for the tests. BTW: doCheck is always disabled when cross-compiling: 
https://github.com/NixOS/nixpkgs/blob/af5ba65b9f4b9fee755fb6701400b7d61d1804e3/pkgs/stdenv/generic/make-derivation.nix#L103

Reply:
Fixed as well :)